amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H

```diff
@@ -330,6 +331,14 @@ struct UpdatePosOp<TurbineFast, SrcTrait>
             vel_pos[i].y() = static_cast<amrex::Real>(pyvel[i]) + bp.y();
             vel_pos[i].z() = static_cast<amrex::Real>(pzvel[i]) + bp.z();
         }
+
+        auto& meta = data.meta();
+        if (!meta.fllc_use || !meta.fllc_is_init) return;
+
+        // Implement the FLLC
+        FLLCOpTurbineBase<TurbineFast, SrcTrait> op(data);
```

Comment:
we should think about where this operator should live/how it is instantiated a little more.  Here you are reconstructing it at every timestep.  This makes it harder to cache the correction and apply it across timesteps like we did in nalu-wind.